packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -82,35 +81,39 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     bool internal subgameAtRootResolved;
 
     /// @notice Semantic version.
-    /// @custom:semver 0.0.13
-    string public constant version = "0.0.13";
+    /// @custom:semver 0.0.19
+    string public constant version = "0.0.19";
```

Comment:
What's the reason for bumping by 6 patch versions?

Reply:
Also not from this PR, diff just from manual merge with the old contract.

Reply:
Ah gotcha, thanks for the clarification